### PR TITLE
add fork_test file

### PR DESCRIPTION
**Pull request by yuzhitao2018**
i can help you to progit

### CODE REVIEW

File: fork_test [hunk 1]
```diff
@@ -0,0 +1 @@
+fork_test file
```
Comment (yuzhitao2018) [inline fork_test:1]: modify the content

Comment (yuzhitao2017) [reply fork_test:1]: change the content to helping

**Comment by yuzhitao2017** [comment]
jkglgl

File: fork_test (hunk 1 above)
Comment (yuzhitao2017) [reply fork_test:1]: hi

**Pull request merged**